packages/terra-avatar/package.json

```diff
@@ -29,7 +29,6 @@
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
     "terra-doc-template": "^2.5.0",
-    "terra-form-select": "^5.9.0",
```

Comment:
There are a couple examples in each of these packages that are using the terra-form-select that will need updated to use the native select.

Reply:
Yup, that's the plan. Was just pushing the dependency removal first, sorry for the confusion!